cypress/tests/overview/top-consumers-tab.cy.ts

```diff
@@ -0,0 +1,25 @@
+import { topConsumerCard } from '../../views/overview';
+import { Perspective, switchPerspective } from '../../views/perspective';
+import { tab } from '../../views/tab';
+
+describe('Test Virtualization Top consumer tab', () => {
+  before(() => {
+    cy.visit('');
+    switchPerspective(Perspective.Administrator);
+  });
+
+  it('ID(CNV-9295) check data available for cpu, memory and storage', () => {
+    cy.visitOverview();
+    tab.navigateToTopConsumers();
+    cy.get(topConsumerCard).eq(0).should('not.contain', 'No data available');
```

Comment:
suggestion: since .eq(0) eq(1) etc are not readable for someone who didn't wrote this test, I want to suggest creating an enum to have more understanding which exact part we testing
e.g if 0 is the title or body or something else of the top consumer card.